### PR TITLE
Add new case of memory and replace logging with test.log

### PR DESCRIPTION
**Pull request by chloerh**
- Check the memory balloon size after restoring a managed save guest

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by chloerh** [comment]
Test result:

File: libvirt/tests/cfg/memory/memory_misc.cfg [hunk 1]
```diff
@@ -98,3 +98,18 @@
...
+                    vmxml_max_mem_rt_unit = 'KiB'
+                    vmxml_max_mem_rt = 55600128
+                    vmxml_memory_unit = 'KiB'
+                    vmxml_memory = 3145728
+                    vmxml_current_mem_unit = 'KiB'
+                    vmxml_current_mem = 3045728
+                    vmxml_vcpu = 4
+                    numa_node_size = 2048000
+                    cpu_attrs = {'numa_cell': [{'id': '0', 'cpus': '0,2', 'memory': '${numa_node_size}', 'unit': 'KiB'}, {'id': '1', 'cpus': '1,3', 'memory': '${numa_node_size}', 'unit': 'KiB'}]}
+
```
Comment (Yingshun) [inline libvirt/tests/cfg/memory/memory_misc.cfg:114]: Should we remove one of the blank line here?

**Review by Yingshun — COMMENTED** [review]
LGTM

File: libvirt/tests/cfg/memory/memory_misc.cfg (hunk 1 above)
Comment (chloerh) [reply libvirt/tests/cfg/memory/memory_misc.cfg]: Yes, removed. Thanks!

**Review by Yingshun — APPROVED** [review]

**Pull request merged**